src/core.rs

```diff
@@ -639,6 +664,26 @@ impl<T> Webauthn<T> {
         self.generate_challenge_authenticate_options(creds, None)
     }
 
+    /// Authenticate a single credential, with the ability to override the userVerification
+    /// policy requested, or extensions in use.
+    ///
+    /// NOTE: Over-riding the UserVerificationPolicy may have SECURITY consequences. You should
+    /// understand how this interacts with the single credential in use, and how that may impact
+    /// your system security.
+    ///
+    /// If in doubt, do NOT use this function!
+    pub fn generate_challenge_authenticate_credential(
+        &self,
+        cred: Credential,
+        policy: UserVerificationPolicy,
```

Comment:
why would we allow overriding the policy? this seems like an easy footgun

Reply:
I seem to remember you requested the ability to override the UV policy for a singular credential for some use cases :) That's why I added it because I think for a single credential, it's still a footgun, but it's not an automatic foot chainsaw like policy on a set of credentials is.

Reply:
Perhaps I misread, but I thought I saw that in `verify_credential_internal` it would error if mismatched.
I think that the right way to "override" it is to just change it on the credential itself.

Reply:
nvm ignore this!

Reply:
Yeah, when you set the override we don't do the policy consistency check. The only thing that *could* occur, is that the credential will verify under discouraged which we have a seperate flag for checking :)